ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -0,0 +1,66 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy.functional.backends.torch as ivy_torch
+
+
+# flip
+@given(
+    dtype_value=

```

Comment:
Please try to make use of the `dtype_values_axis` helper instead of separately using `dtype_and_values` and `get_axis`.